### ISSUE TITLE
Code sample for date range picker results in 508 non-compliance

### ISSUE THREAD

**Issue by bmcorum** [uswds/uswds-site#1271]
## Description

Noticed that both the component preview and component code section results in a 508 compliance error due to the misusage of the aria-describedby attribute and a missing aria-labeledby attribute.

The input tag should be
`<input
        class="usa-input"
        id="event-date-start"
        name="event-date-start"
        type="text"
        aria-labeledby="event-date-start-label"
        aria-describedby="event-date-start-hint"
      />`

## Steps to reproduce the issue

N/A

## Additional information [optional]
![image](https://user-images.githubusercontent.com/1165615/134227885-e8aef8b3-f24e-4893-84ca-122ba501be82.png)
